samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -58,6 +59,28 @@ def training_input(input_name, s3_uri):
     }
 
 
+def model_input(train_image, model_artifact_url):
+    return {
+        "containerHostname": "mnist-kmeans",
+        "image": train_image,
+        "mode": "SingleModel",
+        "modelDataURL": model_artifact_url,
+    }
+
+
+def get_prod_variants(model_name):
```

Comment:
use full name since its a sample to make it easy for new users

add details and links where possible